ui/aura/mus/window_tree_client.cc

```diff
@@ -1193,7 +1193,7 @@ void WindowTreeClient::OnNewBoundsFromHostServer(Id window_id,
 
   // WindowTreeHost expects bounds to be in pixels.
   WindowTreeHostMus* host = GetWindowTreeHostMus(window);
-  host->SetBoundsInPixels(new_bounds);
+  host->SetBoundsFromServer(new_bounds);
```

Comment:
you are right. only updating this bit does not allow to see the bigger picture. I will clean up and update the rest of the work in one shot.